internal/release/atlas/main.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 MongoDB Inc
```

Comment:
do we really need to duplicate this file? a lot of the logic seems to be the same just different package name

Reply:
I think it makes sense since we have some more differences:

- package name
- feedFilename
- ManualLink: to be updated once the doc is ready
- ReleaseNotesLink: to be updated once the doc is ready
- TutorialLink: to be updated once the doc is ready

Reply:
all your points there seem like premature optimisation to me, how different would they really be? And the main reason on why I ask, the same way way we pass version as an argument to this script, why are we not able to pass some of those values as arguments as well and keep the single main? not only arguments, you can also use https://github.com/spf13/pflag which we already include as a dependency as part of cobra

Reply:
updated